README.md

```diff
@@ -39,7 +39,8 @@ The easiest way to start all of them is to use the provided
 docker-compose configuration:
 
 ```sh
-$ docker-compose up -d
+$ docker-compose down --remove-orphans  
```

Comment:
This can be done in a single command with a combination of these (all supported by `up`):

* `--remove-orphans`: removes no longer needed containers
* `--force-recreate`: recreate the containers
* `-V`: recreate the volumes